docs/excel/custom-functions-debugging.md

```diff
@@ -0,0 +1,157 @@
+---
+ms.date: 02/26/2019
+description: Debug your custom functions in Excel.
+title: Custom functions debugging (preview)
+localization_priority: Normal
+---
+# Custom functions debugging (preview)
```

Comment:
i think we can structure the article a bit different.  I will clearly separate the platform where the debugging starts. 

1. Debugging in Excel for Windows.  then clearly separate you have 2 options here: directly in VS Code and using the Browser dev tools [this last will be a link because is also used for Mac].
2. Debuggingin  Excel Online :  using VSCode and Edge.
3. debuggin in Excel for  Mac -> provide a link to the debugging in browser
4. Debugging if not using VSCode via command lines.